workflow/rules/common.smk

```diff
@@ -55,6 +55,43 @@ def type_generator(types):
     else:
         return types
 
+def get_bam_input(wildcards, t_n=None, use_sample_wildcard=True):
+    if use_sample_wildcard is True and t_n is None:
+        sample_str = "{}_{}".format(wildcards.sample, wildcards.type)
+    elif use_sample_wildcard is True and t_n:
+        sample_str = "{}_{}".format(wildcards.sample, t_n)
+    else:
+        sample_str = wildcards.file
+
```

Comment:
```suggestion
```

Reply:
I'm not sure if it has now decided that this should be different or if I marked in the wrong row before